httpcore5/src/main/java/org/apache/hc/core5/http/HttpStatus.java

```diff
@@ -95,78 +93,70 @@
-    /** {@code 402 Payment Required} (HTTP/1.1 - RFC 2616) */
+    /** {@code 402 Payment Required} (HTTP/1.1 - RFC 7231) */
     int SC_PAYMENT_REQUIRED = 402;
-    /** {@code 403 Forbidden} (HTTP/1.0 - RFC 1945) */
+    /** {@code 403 Forbidden} (HTTP/1.0 - RFC 7231) */
     int SC_FORBIDDEN = 403;
-    /** {@code 404 Not Found} (HTTP/1.0 - RFC 1945) */
+    /** {@code 404 Not Found} (HTTP/1.0 - RFC 7231) */
     int SC_NOT_FOUND = 404;
-    /** {@code 405 Method Not Allowed} (HTTP/1.1 - RFC 2616) */
+    /** {@code 405 Method Not Allowed} (HTTP/1.1 - RFC 7231) */
     int SC_METHOD_NOT_ALLOWED = 405;
-    /** {@code 406 Not Acceptable} (HTTP/1.1 - RFC 2616) */
+    /** {@code 406 Not Acceptable} (HTTP/1.1 - RFC 7231) */
     int SC_NOT_ACCEPTABLE = 406;
-    /** {@code 407 Proxy Authentication Required} (HTTP/1.1 - RFC 2616)*/
+    /** {@code 407 Proxy Authentication Required} (HTTP/1.1 - RFC 7231)*/
     int SC_PROXY_AUTHENTICATION_REQUIRED = 407;
-    /** {@code 408 Request Timeout} (HTTP/1.1 - RFC 2616) */
+    /** {@code 408 Request Timeout} (HTTP/1.1 - RFC 7231) */
     int SC_REQUEST_TIMEOUT = 408;
-    /** {@code 409 Conflict} (HTTP/1.1 - RFC 2616) */
+    /** {@code 409 Conflict} (HTTP/1.1 - RFC 7231) */
     int SC_CONFLICT = 409;
-    /** {@code 410 Gone} (HTTP/1.1 - RFC 2616) */
+    /** {@code 410 Gone} (HTTP/1.1 - RFC 7231) */
     int SC_GONE = 410;
-    /** {@code 411 Length Required} (HTTP/1.1 - RFC 2616) */
+    /** {@code 411 Length Required} (HTTP/1.1 - RFC 7231) */
     int SC_LENGTH_REQUIRED = 411;
-    /** {@code 412 Precondition Failed} (HTTP/1.1 - RFC 2616) */
+    /** {@code 412 Precondition Failed} (HTTP/1.1 - RFC 7231) */
     int SC_PRECONDITION_FAILED = 412;
-    /** {@code 413 Request Entity Too Large} (HTTP/1.1 - RFC 2616) */
+    /** {@code 413 Request Entity Too Large} (HTTP/1.1 - RFC 7231) */
     int SC_REQUEST_TOO_LONG = 413;
-    /** {@code 414 Request-URI Too Long} (HTTP/1.1 - RFC 2616) */
+    /** {@code 414 Request-URI Too Long} (HTTP/1.1 - RFC 7231) */
     int SC_REQUEST_URI_TOO_LONG = 414;
-    /** {@code 415 Unsupported Media Type} (HTTP/1.1 - RFC 2616) */
+    /** {@code 415 Unsupported Media Type} (HTTP/1.1 - RFC 7231) */
     int SC_UNSUPPORTED_MEDIA_TYPE = 415;
-    /** {@code 416 Requested Range Not Satisfiable} (HTTP/1.1 - RFC 2616) */
+    /** {@code 416 Requested Range Not Satisfiable} (HTTP/1.1 - RFC 7231) */
     int SC_REQUESTED_RANGE_NOT_SATISFIABLE = 416;
-    /** {@code 417 Expectation Failed} (HTTP/1.1 - RFC 2616) */
+    /** {@code 417 Expectation Failed} (HTTP/1.1 - RFC 7231) */
     int SC_EXPECTATION_FAILED = 417;
     /** {@code 421 Misdirected Request} (HTTP/2 - RFC 7540) */
     int SC_MISDIRECTED_REQUEST = 421;
 
-    /**
-     * Static constant for a 418 error.
-     * {@code 418 Unprocessable Entity} (WebDAV drafts?)
-     * or {@code 418 Reauthentication Required} (HTTP/1.1 drafts?)
-     */
-    // not used
-    // int SC_UNPROCESSABLE_ENTITY = 418;
```

Comment:
Since you have removed that, I do not see https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422 anywhere.